proposals/0029-cooperative-vector.md

```diff
@@ -250,7 +250,7 @@ row/column of the matrix is valid memory.
 The **matrix stride** is 16-byte aligned.
 
 This operation doesn't perform bounds checking for matrix loads. If any part of
-the matrix load is out of bounds then the entire operation is undefined.
+the matrix load is out of bounds then the entire matrix load will return zero.
```

Comment:
This shouldn't say "doesn't perform bounds checking"
Instead, it should say something like:

```
When the **matrix resource** has bounds checking, bounds are only checked at
matrix granularity. If any part of the matrix load is out of bounds, then the
entire matrix load will return zero.

If the **matrix resource** does not have bounds checking, such as when it is
supplied through a root descriptor, and any part of the matrix load is out of
bounds, the operation is undefined.
```

And similar for bias load, and the corresponding store operations.